test/e2e/scheduling/preemption.go

```diff
@@ -200,7 +202,9 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 			gomega.Expect(livePod.DeletionTimestamp).To(gomega.BeNil())
 		}
 
-		framework.ExpectEqual(podPreempted, true)
+		if !podPreempted {
```

Comment:
for readability, prefer to move it under L198.

Reply:
Done!